### ISSUE TITLE
[BUG]: push:mysql trying to truncate table with boolean 

### ISSUE THREAD

**Issue by TiltedToast** [drizzle-team/drizzle-kit-mirror#59]
I've been playing around a bit with the new `push:mysql` command and noticed that it's trying to wipe my table any time i run it because it's supposedly trying to change the type to 'boolean' for some reason.

Here's the message and corresponding table schema:

```
· You're about to change over_18 column type from alter_table_alter_column_set_type to boolean with 108 items
```

```ts
export const redditPosts = mysqlTable(
    "reddit_posts",
    {
        id: int("id").autoincrement().primaryKey(),
        subreddit: varchar("subreddit", { length: 50 }).notNull(),
        title: varchar("title", { length: 255 }).notNull(),
        url: varchar("url", { length: 255 }).notNull(),
        over_18: boolean("over_18").notNull(),
        permalink: varchar("permalink", { length: 255 }).notNull(),
    },
    (table) => {
        return {
            subredditIdx: index("subreddit_idx").on(table.subreddit),
        };
    }
);
```

I'd assume it's breaking because mysql doesn't actually have a boolean type and it's trying to turn the existing tinyint into a boolean? Doesn't end up actually changing anything apart from wiping the table clean. Not really the end of the world for me, but still pretty annoying having to wipe local data to change the schema.

**Comment by kylewardnz** [comment]
I posted the same issue in the Discord about 20 minutes before this. Downgrading back to `0.17.6-76e73f3` seems to work around it for me.

**Comment by AndriiSherman** [comment]
Looking into it, thanks!

**Comment by natedunn** [comment]
Oof yeah can confirm this issue on my end. Wipes whole table.

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.18.1`
